src/main/java/org/gitlab4j/api/models/MergeRequest.java

```diff
@@ -15,6 +15,7 @@
     private Integer approvalsBeforeMerge;
     private Assignee assignee;
     private List<Assignee> assignees;
+    private List<Assignee> reviewers;
```

Comment:
Probably a good thing to define a new class `Reviewer` that extends from `AbstractUser` for this?

Reply:
I thought about it, it just felt redundant to create an identical class but would be more in keeping with the rest of the system